awsapilib/controltower/controltower.py

```diff
@@ -806,12 +806,14 @@ def get_changing_accounts(self):
             accounts (Account): A list of under change account objects under control tower's control.
 
         """
-        products = self.service_catalog.search_provisioned_products()
+        changing_products = self.service_catalog.search_provisioned_products(
+            Filters={
+                "SearchQuery":["status:UNDER_CHANGE type:CONTROL_TOWER_ACCOUNT"]
```

Comment:
Would be great if the filter was part of the configuration constants so it's update would be easy if needed without having to visit the code

Reply:
The whole filter as a config item? or just he query ? 

`changing_products = self.service_catalog.search_provisioned_products(
            Filters={
                "SearchQuery":[PROVISIONED_PRODUCTS_UNDER_CHANGE_FILTER]
            }
        )`
Or:
`changing_products = self.service_catalog.search_provisioned_products(
            Filters=PROVISIONED_PRODUCTS_UNDER_CHANGE_FILTER
        )`

Reply:
went with the query as config item, because based on the docs , the only key supported is `SearchQuery` , so no point having it as a config.

Reply:
Very well said! In that case wouldn't it make more sense to include the list literal since there is no other way to present it?

Reply:
You sir, are a gentleman and a scholar.